gsoc/guideline.md

```diff
@@ -80,6 +80,6 @@ create a file for your organization and edit its contents as appropriate.
 
 Organization logos should be called `ORGANIZATION-logo.png` and be less than 100kB in size.
 
-## HSF GSoC Mentor Guideline 2021
+## HSF GSoC Mentor Guideline 2022
```

Comment:
We should probably add below a description of the new properties that we want proposals to contain (duration, difficulty, ...)

Reply:
into the text or just into the google doc?

Reply:
Just in the doc containing the instructions